x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/logical/join/InlineJoin.java

```diff
@@ -146,33 +153,44 @@ public record LogicalPlanTuple(LogicalPlan stubReplacedSubPlan, LogicalPlan orig
      * Aggregate[[],[MAX(x{r}#99,true[BOOLEAN]) AS y#102]]
      * \_Limit[1000[INTEGER],false]
      *   \_LocalRelation[[x{r}#99],[IntVectorBlock[vector=ConstantIntVector[positions=1, value=1]]]]
```

Comment:
This javadoc might require an update to be inline with the added paragraph, but this is minor, can remain as is.